src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -42,10 +45,15 @@ export class Artwork extends React.Component<Props> {
     }
 
     sections.push("details")
-
     sections.push("aboutArtist")
 
-    sections.push("otherWorks")
+    if (
```

Comment:
This logic in particular is only going to get more complex as we build out support for more contexts so I worry about having to add to this logic here.

What's the risk of always pushing on this section?

Reply:
We don't want to push any section if there's no supporting data. I think we should keep adding checks here and see how things land, and we can always move logic to metaphysics if necessary.

Reply:
But what happens if we do? I'm curious what the impact will be on the added separator etc. since this is the last section.

Reply:
There will be an extra separator rendered at the bottom of the page.

You can see it doesn't need to get too crazy: https://github.com/artsy/emission/blob/master/src/lib/Scenes/Show/Screens/Detail.tsx